tilda.appdata.xml

```diff
@@ -38,6 +38,5 @@
   </screenshot>
  </screenshots>
  <url type="homepage">https://github.com/lanoxx/tilda</url>
- <updatecontact>lanoxx_at_gmx.net</updatecontact>
- <project_group>Tilda</project_group>
+ <update_contact>lanoxx_at_gmx.net</update_contact>
```

Comment:
According to the specification developer Matthias Klumpp the value is is case-insensitive, so I am fine with this change.